### DISCUSSION TITLE
shortcut, config and import-config options do not seem to work (user level only)

### DISCUSSION THREAD

**Discussion by sebastianschauenburg** [rustdesk/rustdesk#6391] [General]
### Bug Description

The options `--config` and `--import-config` do not seem to work (or not consistently). Also adding information to the Windows shortcut also does not seem to work as [described in the section "Put config in rustdesk.exe file name (Windows only) "](https://rustdesk.com/docs/en/self-host/client-configuration/)

On Linux (AppImage) with the `--config` option, the following message is shown:
"Installation and administrative privileges required!"

On Windows (exe, not installed, only run) with either `--config` or `--import-config` or the information in the shortcut, no error is shown.

In both cases, if I check the rustdesk config (once I start rustdesk without options) I see no custom settings in `Settings -> Network`.

Note: I've used a self hosted rustdesk instance (the ID server works like a charm) and I've used the 'export' button to create the configuration string.

### How to Reproduce

- use a custom rustdesk server and export the configuration string
  - try to run/use the configuration string with `--config` or `--import-config`
  - try to create a custom Windows shortcut
- run rustdesk and check the settings

### Expected Behavior

I would've expected the custom settings to have been imported and set (and visible in `Settings -> Network`).

On Linux I would not expect to require elevated privileges for this user level setting.

On Windows  I would've expected an error message why the application would not start.

### Operating system(s) on local side and remote side

Server: Linux,  Client: Linux and Windows

### RustDesk Version(s) on local side and remote side

1.2.3

### Screenshots

![rustdesk-windows-config](https://github.com/rustdesk/rustdesk/assets/8626247/965f3dd2-012f-4b84-8354-8c0a769f3fd8)

### Additional Context

The reason for me trying this, is so that I can provide someone the .exe with the custom settings preconfigured, so they only need to run rustdesk (without having to install it even). I don't mind creating a custom shortcut or .bat script to include the settings there.

I've also seen bug [https://github.com/rustdesk/rustdesk/issues/5740] but it seems different.

**Comment by fufesou** [comment]
You should install rustdesk, and run as Administrator to update the configuration.

**Reply by sebastianschauenburg** [reply]
Is that the way it is supposed to work? (e.g. by design) 
(because if that is so, then perhaps I could update the documentation to point that out)

And would this behavior perhaps be enhanced (to the way I assumed it would work, without elevated privileges) if there would be more interest or something like that?

**Comment by hatl** [comment]
also: the suggested Linux deployment is not working when using the appimage version:
https://rustdesk.com/docs/en/self-host/client-deployment/
